sdk/netapp/arm-netapp/src/models/mappers.ts

```diff
@@ -1881,6 +1881,17 @@ export const VolumePatch: coreClient.CompositeMapper = {
           className: "VolumePatchPropertiesExportPolicy",
         },
       },
+      protocolTypes: {
```

Comment:
@kazrael2119 could you share the swagger pr with me?

@josefree  do we allow this to update the change in the same api version?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/29206

Reply:
@josefree  confirmed with service team here https://github.com/Azure/azure-rest-api-specs/pull/29206#discussion_r1692521707, it seems a bugfix for spec. I may prefer not blocking this release. Let me know if any concern.